cms/templates/product_page.html

```diff
@@ -65,26 +65,32 @@
 
 {% block content %}
 <div>
-    {% include "partials/hero.html" %}
-    {% include "partials/metadata-tiles.html" %}
-    {% include "partials/subnav.html" with product=page.product %}
-    {% if page.outcomes %} {% include "partials/learning-outcomes.html" with page=page.outcomes %} {% endif %}
-    {% if page.who_should_enroll %} {% include "partials/target-audience.html" with page=page.who_should_enroll %} {% endif %}
-    {% if page.techniques %} {% include "partials/learning-techniques.html" with page=page.techniques %} {% endif %}
-    {% if page.testimonials %} {% include "partials/testimonial-carousel.html" with page=page.testimonials %} {% endif %}
-    {% if page.faculty %} {% include "partials/faculty-carousel.html" with page=page.faculty %} {% endif %}
-    {% if page.course_lineup and page.course_pages %}
-      {% with program_page=page.program_page courseware_pages=page.course_pages %}
-        {% pageurl program_page as program_url %}
-        {% if page.program %}
-          {% include "partials/course-carousel.html" with page=page.course_lineup %}
-        {% else %}
-          {% include "partials/course-carousel.html" with page=page.course_lineup button_title="View Full Program" button_url=program_url %}
-        {% endif %}
+    {% with background_video_url=page.background_video_url testimonials=page.testimonials faculty=page.faculty  %}
```

Comment:
are we utilizing the var `background_video_url` ?

Reply:
This is being used in `hero`